src/components/Timeline/package.json

```diff
@@ -18,15 +18,24 @@
         "dist"
     ],
     "scripts": {
-        "build": "tsc -b",
+        "build": "tsc -b && copyfiles -u 1 \"src/**/*.css\" dist",
         "clean": "yarn rimraf dist tsconfig.tsbuildinfo"
     },
     "dependencies": {
         "@gemeente-denhaag/baseprops": "^0.1.7",
-        "@material-ui/core": "^4.11.0"
+        "@gemeente-denhaag/design-tokens-common": "^0.1.1",
+        "@gemeente-denhaag/design-tokens-components": "^0.1.1",
+        "@gemeente-denhaag/design-tokens-proprietary": "^0.1.1",
```

Comment:
These should not be added here, but installed separately by user of the library.
```suggestion
```